include/superior_mysqlpp/logging.hpp

```diff
@@ -212,14 +222,23 @@ namespace SuperiorMySqlpp {
             virtual void logSharedPtrPoolHealthCareJobError(std::uint_fast64_t /*poolId*/, const std::exception&) const noexcept override {}
             virtual void logSharedPtrPoolHealthCareJobError(std::uint_fast64_t /*poolId*/) const noexcept override {}
 
-            virtual void logSharedPtrPoolDnsAwarePoolManagementCycleStart(std::uint_fast64_t /*poolId*/) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementChangeDetected(std::uint_fast64_t /*poolId*/) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementCheckError(std::uint_fast64_t /*poolId*/, std::exception&) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementCheckError(std::uint_fast64_t /*poolId*/) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementCycleEnd(std::uint_fast64_t /*poolId*/) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementStopped(std::uint_fast64_t /*poolId*/) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementError(std::uint_fast64_t /*poolId*/, std::exception&) const noexcept override {}
-            virtual void logSharedPtrPoolDnsAwarePoolManagementError(std::uint_fast64_t /*poolId*/) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementCycleStart(std::uint_fast64_t /*poolId*/) const noexcept override {}
```

Comment:
deprecated ma variantu kde muzes dat message, premyslim jestli by stalo za to rict co to nahrazuje - nevyhoda je jen, ze je ta informace duplikovana

Reply:
Taky jsem o tom premyslel, IMO to nestoji za to, muzem se o tom pobavit

Reply:
I'd at least add a single comment above them then - again, explicit is better then implicit.

Reply:
ACK